### PR TITLE
add collabora-code chart

### PR DESCRIPTION
**Pull request by chrisingenhaag**
Signed-off-by: Christian Ingenhaag <christian.ingenhaag@googlemail.com>

#### What this PR does / why we need it:

Adds easy deployment for collabora online office suite developer edition in order to run is in kubernetes.

#### Special notes for your reviewer:

* Additionally to the OK message given at port 9980 you should be able to log in at http://localhost:9980/loleaflet/dist/admin/admin.html with the default credentials from values.yaml
* Valid SSL Termination is kind of mandatory for a fully working integation of collabora with a wopi compatible solution like nextcloud for example. But it´s user dependent which ingress system one wants to use or if an kubernetes-external ssl-termination is provided

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by chrisingenhaag** [comment]
I will try to assign someone else as the PR did not get any review until now. Review welcome, I´ll add OWNERS file tonight in order to need review just once.

/assign @unguiculus

File: stable/collabora-code/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Introduction
+
+This chart creates a single Collabora CODE Pod to run Collabora CODE suite, for example as integration together with nextcloud. Installation is based on the docker documentation [CollaboraDocker](https://www.collaboraoffice.com/code/docker/).
+
+For most easy integration it´s recommended to use cert-manager together with your favorite ingress controller to get a fully working, ssl-terminated Collabora CODE server.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (mattfarina) [inline stable/collabora-code/README.md]: Can you please update this to Kubernetes 1.9+ as that is when the `apps/v1` API used below became available.

**Review by mattfarina — COMMENTED** [review]
Just one tiny nit.

File: stable/collabora-code/README.md (hunk 1 above)
Comment (chrisingenhaag) [reply stable/collabora-code/README.md]: Yes, of course. I fixed the README.

**Review by mattfarina — APPROVED** [review]

**Comment by mattfarina** [comment]
@chrisingenhaag There appears to be some problem with the chart. Can you take a look.

**Comment by chrisingenhaag** [comment]
@mattfarina I checked some improvements around port naming, but had some trouble with testing it locally. Now it should be working, at least my local tests pass. But I cannot call /retest by myself.

Can you please trigger the test?

**Comment by aleksey005** [comment]
Not work with nextcloud plugin - because it violates the following Content Security Policy directive
image

**Comment by chrisingenhaag** [comment]
> Not work with nextcloud plugin - because it violates the following Content Security Policy directive
> image
> image

Actually it does work with nextcloud plugin. 
How do you access your collabora instance? Did you setup an ingress? How do you access your nextcloud instance?

Afaik ....svc.cluster.local is made for k8s cluster-internal communication only. From your screenshot your collabora instance runs under domain collabora-code.collab, but you defined collabora-code.collabora-code.svc.cluster.local as helm value collabora.server_name. This has to match.

I´m preparing a blog post on this for a sample setup. I´ll post the link here as soon it is ready.

**Comment by aleksey005** [comment]
How do you access your collabora instance?>
I set ingress https real (nextcloud.doman.com and collabora.domain.com) domain names for everyone and tied them. 

Did you setup an ingress?>
ingress for collabora - message OK for https://collabora.domain.com

How do you access your nextcloud instance?>
Access for nextcloud over ingress https with real domani name

Maybe I made a mistake and should not be addressed by cluster names, but by real names also does not work, if I specify them in the plugin settings.

**Pull request merged**